builds/e2e/nested-longhaul.yaml

```diff
@@ -1,5 +1,12 @@
 trigger: none
 pr: none
+schedules:
+- cron: "0 23 * * 4"
```

Comment:
I guess this nested longhaul pipeline doesn't share the same agents as the single-node longhaul, so it's ok to use the same schedule?

Reply:
That's correct. This is the schedule we're already using today, I just copied it to the main branch as well.